src/reduceChildren.test.js

```diff
@@ -0,0 +1,50 @@
+const reduce = require('./reduceChildren')
+describe('reduceChildren', () => {
+  it('should return all texts', () => {
+    const data = [
+      {
+        type: 'text',
+        value: 'Install ',
+        position: {
+          start: { line: 10, column: 4, offset: 413 },
+          end: { line: 10, column: 12, offset: 421 },
+          indent: []
+        }
+      },
+      {
+        type: 'inlineCode',
+        value: 'webpack',
+        position: {
+          start: { line: 10, column: 12, offset: 421 },
+          end: { line: 10, column: 21, offset: 430 },
+          indent: []
+        }
+      }
+    ]
+    expect(reduce(data)).toEqual(['Install ', 'webpack'])
+  })
+
+  it('should return all texts within inline code', () => {
```

Comment:
When having `()` it also get included in the value?

Reply:
`()` alone isn't special to remark, so it's parsed as `text`.

Reply:
Cool